hack/replace-nto.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -e
+
+# This script assumes a deployed OpenShift cluster and an environment .
+# Please note cvo_scale_down() scales down CVO replica to 0 to prevent it from
+# overriding the custom NTO image.  If you wish to upgrade your cluster,
+# do not use this script.
+
+
+# Scale CVO down not to overwrite custom NTO.
+cvo_scale() {
+  local replicas=${1:-0}
+  oc scale deploy/cluster-version-operator --replicas=$replicas -n openshift-cluster-version
+}
+
+nto_deploy_custom() {
+  oc project openshift-cluster-node-tuning-operator
+  for file in ./manifests/*; do
+    # We will skip the operator deployment manifest and just patch it instead. 
+    if [[ "$file" != ./manifests/50-operator* && "$file" == *.yaml ]]; then
```

Comment:
Why do we want to skip that manifest?

Reply:
This manifest includes the operator deployment (which is already deployed with OCP). we are patching the image fields and inserting our custom image reference instead of changing and reapplying the yaml (supposed to have the same effect).

Reply:
>  (which is already deployed with OCP).

All the manifests are already deployed with OCP, the operator manifest is no way special and the developer might have changed it to suit his/her own needs.

Reply:
I agree. I think then we should only skip the `50-operator-ibm-cloud-managed.yaml` then.